src/app-gocardless/bank-factory.js

```diff
@@ -45,6 +46,7 @@ export const banks = [
   SpkKarlsruhekarsde66,
   VirginNrnbgb22,
   Berliner_Sparkasse_beladebexxx,
+  KBCkredbebb,
```

Comment:
Same with this one please, again - feel free to move `Berliner_Sparkasse_beladebexxx` at the same time.